### PR TITLE
Re-added Realistic F1 DRAM machine

### PR DESCRIPTION
**Pull request by drichmond**

### CODE REVIEW

File: testbenches/Makefile [hunk 1]
```diff
@@ -60,7 +60,7 @@ include $(REGRESSION_PATH)/targets.mk
 regression: regression.log
 
 regression.log: $(TARGETS)
-	@cat $(foreach tgt,$(TARGETS),$(tgt)/regression.log) > regression.log
```
Comment (mrutt92) [inline testbenches/Makefile]: meant to be in this PR?

Comment (drichmond) [reply testbenches/Makefile]: Disappears with a rebase

Comment (drichmond) [reply testbenches/Makefile]: (Which I just did)

**Comment by drichmond** [comment]
I'm not sure I want to merge this PR. I might just leave it here until I implement something to remove it from the regression machines list. 

(It just takes forever to run...)

**Comment by mrutt92** [comment]
k. you can use the filter-out function makefile function to remove it from that list?

**Comment by drichmond** [comment]
Yes, fixed

**Pull request merged**